Kernel/Graphics/Intel/DisplayConnectorGroup.cpp

```diff
@@ -249,6 +249,9 @@ ErrorOr<void> IntelDisplayConnectorGroup::set_mode_setting(IntelNativeDisplayCon
 {
     SpinlockLocker locker(connector.m_modeset_lock);
 
+    VERIFY(to_underlying(connector.connector_index()) < m_connectors.size());
+    VERIFY(const_cast<IntelNativeDisplayConnector*>(&connector) == m_connectors[to_underlying(connector.connector_index())].ptr());
```

Comment:
I doubt the const cast is required here to compare a pointer to to a const pointer